roles/opensrp/meta/main.yml

```diff
@@ -19,9 +26,24 @@ dependencies: # noqa 701
   tomcat_max_filesize: "{{ opensrp_tomcat_max_filesize }}"
   tomcat_fsize_threshold: "{{ opensrp_tomcat_fsize_threshold }}"
   tomcat_ajp_port: "{{ opensrp_tomcat_ajp_port }}"
+  when: opensrp_version is match('v1.\d.\d.\w+')
```

Comment:
noting that not working on branches will affect this and line 43 below too

Reply:
@pld @jasonrogena I have changed the approach and will use a variable `opensrp_tomcat_version` to control which version is installed